### ISSUE TITLE
scrolling lags when realtive-numbers is enabled

### ISSUE THREAD

**Issue by marcelhadorn** [justmoon/relative-numbers#47]
Hi,

I experience a strong lag and not smooth behaviour when scrolling with relative-numbers enabled. I really like realtive-numbers alongside vim-mode-plus, but it kind of causes motion-sickness to me when its enabled due the laggy scrolling I experience. I'm on a MacPro with OSX El Cap.
I could determine, that it's caused by realtive-numbers by deactivating one package after the other till the lag was gone. Anyone else experiencing this?

**Comment by mindbound** [comment]
I'm also having this, even in normal (non-Vim) mode.

**Comment by marcelhadorn** [comment]
@mindbound yeah it's definitely related to relative numbers and not to vim-mode-plus

**Comment by justmoon** [comment]
Butter-smooth for me here on Linux. Anyone with a Mac wanna take a stab at a PR, please go for it!
